src/commands/ls.ts

```diff
@@ -40,57 +40,56 @@ const lsArgs = <T>(yargs: yargs.Argv<T>) =>
       string: true,
       default: [] as string[],
     })
+    .option("size", {
```

Comment:
this lets us intercept the `--size` flag, and re-add it into `arguments` before sending it to the api backend without having to do any special text parsing.